### DISCUSSION TITLE
c-ray on AROS

### DISCUSSION THREAD

**Discussion by mattrust** [VKoskiv/c-ray#114] [Show and tell]
This screenshots shows c-ray running under the AROS operating system.

![grafik](https://user-images.githubusercontent.com/18250926/234119475-45610b40-55c6-4383-89da-d245a3cae65b.png)

**Comment by vkoskiv** [comment]
VERY cool! I love seeing it run on new platforms, thanks for showing this!
What sort of hardware are you running it on?

**Reply by vkoskiv** [reply]
Also, you can find more scenes provided with the repo: `ls input/*.json`. `hdr.json` is my favourite test scene. I also have more complex scenes I can give you directly, like the dining room scene shown in the README.

**Comment by mattrust** [comment]
CPU is Intel Core i7-4770 @ 3.4 GHz
Since AROS is a clone of an operating system from about 1993 it supports only one core. (There is an experimental SMP version which isn't supported by pthread yet).
I've rendered hdr.json with the Linux version of c-ray. It's quite impressive.
